karton_whatvpn/karton_whatvpn.py

```diff
@@ -0,0 +1,67 @@
+import subprocess
+
+from artemis import utils, load_risk_class
+from artemis.binds import TaskStatus, TaskType, Service
+from artemis.module_base import ArtemisBase
+from artemis.task_utils import get_target_url, get_target_host
+from karton.core import Task
+import string
+
+from extra_modules_config import ExtraModulesConfig
+
+logger = utils.build_logger(__name__)
+
+@load_risk_class.load_risk_class(load_risk_class.LoadRiskClass.LOW)
+class WhatVPN(ArtemisBase):
+    """ 
+    Runs what-vpn -> SSL VPN identifier
+    """
+    
+    identity = "what-vpn"
+    filters = [
+        {"type": TaskType.IP.value}
```

Comment:
and why not domain? or even SERVICE?

Reply:
I'll stay with IP task type as profit from scanning all discovered ports is debatable while increase number of sent requests  is noticeable.
Might change in the future based on observations.